### ISSUE TITLE
delete m2m instance indexes not update

### ISSUE THREAD

**Issue by zhimingzhang123** [barseghyanartur/django-elasticsearch-dsl-drf#197]
Hi, I have a question:
  I am running this demo, I want test signals, when I delete m2m instance (tags instance), the indexes not update

```python
# signals.py
# If it is `books.Tag` that is being updated.
        if model_name == 'tag':
            instances = instance.books.all()
            print('tag')
            print(instances) # ---> <QuerySet []>
            for _instance in instances:
                registry.update(_instance)
                # registry.delete(_instance, raise_on_error=False)
```
**when I delete m2m instance before**
I want to delete `Trilogy` instance 
```
{
            "id": 54,
            ...
            "tags": [
                "Fantasy",
                "History",
                "Trilogy"   # note this
            ],
        }
```
**when I delete after**
```
{
            "id": 54,
            ...
            "tags": [
                "Fantasy",
                "History",
                "Trilogy"   # note this, already exists
            ],
        }
```
demo from project.

**Comment by zhimingzhang123** [comment]
```
python=3.7
django=2.1
djangorestframework=3.10
django-elasticsearch-dsl-drf=0.20.8
django-elasticsearch-dsl=6.4.2

elasticsearch=6.5.4
```

**Comment by barseghyanartur** [comment]
Could you rephrase your question (in a better English). I don't understand your issue.

**Comment by barseghyanartur** [comment]
Closing this for now. Feel free to request reopening.
